inngest-core/src/test/kotlin/com/inngest/signingkey/BearerTokenKtTest.kt

```diff
@@ -0,0 +1,47 @@
+package com.inngest.signingkey
+
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFailsWith
+
+class BearerTokenKtTest {
+    @Test
+    fun `test invalid signing key format`() {
+        assertFailsWith<InvalidSigningKeyException> {
+            getAuthorizationHeader("signke-prod-blah")
+        }
+    }
+
+    @Test
+    fun `test signing key is not a valid hex string`() {
+        assertFailsWith<InvalidSigningKeyException> {
+            getAuthorizationHeader("signkey-prod-8fjau3mn")
```

Comment:
This was the example key from the spec https://github.com/inngest/inngest/blob/main/docs/SDK_SPEC.md#414-requirements-when-sending-a-request, but it turns out the key also needs to be a valid hex string, so this one will actually fail.